wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireOutput.kt

```diff
@@ -121,6 +121,7 @@ open class CustomOutput @Inject constructor() : WireOutput() {
   var includes: List<String>? = null
   var excludes: List<String>? = null
   var exclusive: Boolean = true
+  var applyDefaultTargetIfNoTargetAssigned = false
```

Comment:
We need doc, or a better name. By the way, how can a custom output _not_ providing a target? Isn't it a target itself?

Reply:
I'm not a fan of the name, let me give it some more thought. 
So it is assumed that the customer handler generate proto classes. Our custom handler doesn't, it purely meant as addition action to be taken, after/during the generating of the proto classes. In most causes the `kotlin` or `java` target is specified but in some causes we depend on the defaults. When the rely on the defaults, the wire plugin runs the custom handler but doesn't generate any proto classes, which isn't surprising.